dice.py

```diff
@@ -3,15 +3,18 @@
 import random
 
 def roll_the_dice(stuff = ""):
-    if stuff[0] == "-":
-        result = "There's no such thing as a negative die, bro."
+    if stuff == "": # Empty string handler
+        result = "Bro, insert some words to get a random choice, or a number for a random number (1-#)"
     else:
-        if stuff.isdigit(): # If the string is all digits, will return true
-            size = int(stuff) # Because stuff is a string, we need a integer
-            if size <= 0:
-                result = "Cannot roll 0, bro. Come on, think of a 0 sided die."
-            else:
-                result = random.randint(1,size) # Random int from 1 to size
-        else: # If not all digits, do other stuff, returns false
-            result = random.choice(stuff.split(" ")) # Uses a space as the split sperator
+        if stuff[0] == "-": # If the first character of string is "-"
+            result = "There's no such thing as a negative die, bro."
+        else:
+            if stuff.isdigit(): # If the string is all digits, will return true
+                size = int(stuff) # Because stuff is a string, we need a integer
+                if size == 0 or 1: # If the integer is 0 or 1
+                    result = "Cannot roll "+str(size)+", bro. Come on, think of a "+str(size)+" sided die."
```

Comment:
this is great! I should point out that string interpolation is preferable to doing "string" + "thing".

ex: `"this is a {0} bot, bro".format("squanchy")` returns `"this is a squanchy bot, bro"`

The number inside the `{ }` indicates what position argument to replace with `.format()` method. You can also use a position more than once. 

ex: `"ants {0} in my {0}  {1} eyes".format(  ( "not" , "fleeb") )`  returns
 `"ants not in my not fleeb eyes"`.

FIX IT :-1: